Gemfile.lock

```diff
@@ -19,6 +19,3 @@ DEPENDENCIES
   hologram (= 1.2.0)
   sass (~> 3.3.14)
   scss-lint (= 0.26.2)
-
```

Comment:
source control is for our code, I too don't like checking in generated code/stuff we didn't write ourselves